docs/ManualDeploymentGuideScalarDLAuditorOnAWS.md

```diff
@@ -0,0 +1,172 @@
+# Deploy Scalar DL Auditor on AWS
+
+Scalar DL Auditor is an optional component that manages the identical states of Ledger and helps Scalar DL clients to detect Byzantine faults.
+This guide shows you how to manually deploy Scalar DL Auditor on a managed database service and a managed Kubernetes service in AWS as part of deploying Scalar DL for production.
+
+## What we create
+
+![image](images/eks-auditor-arch.png)
+
+In this guide, we will create the following components for Auditor.
+
+* A VPC with NAT gateway
+* An EKS cluster with two Kubernetes node groups
+* A managed database service
+    * DynamoDB
+* A Bastion instance with a public IP
+* Amazon CloudWatch
+
+## Prerequisites
+
+* Scalar DL Ledger deployment with Auditor configuration completed.
+
+## Step 1. Create an environment
+
+It is highly recommended to deploy Scalar DL Auditor in a different administrative domain from the one for Ledger in production because a single fully-privileged administrator can do any malicious activities in a single administrative domain.
+However, for ease of explanation, we deploy Auditor in the same administrative domain as Ledger (i.e., a different cluster on the same subscription) in this guide.
+
+You need to create a network, Scalar DL supported database and EKS service in a different administrative domain for Auditor deployment using the steps of creating a Ledger environment.
+
+### Requirements
+
+* You must create a VPC for Auditor with a different IP address range that is not used by Ledger and Client.
+
+### Recommendations
+
+* You should deploy Ledger and Auditor in separate administrative domains.
+
+### Steps
+
+* Follow step 1 to step 3 in the [Deploy Scalar DL on AWS](./ManualDeploymentGuideScalarDLOnAWS.md) guide to create a Scalar DL Auditor environment.
+
+## Step 2. Peer the Virtual Networks
+
+In this guide, Ledger, Auditor, and Client applications are deployed on the private subnet of the different VPCs, 
+so you need to add peering for internal communication between the Auditor, Ledger, and Client application.  
+
+### Requirements
+* You must create peering between the Ledger and Auditor VPCs.
```

Comment:
As you mentioned, these are not requirements. Requirements should be states.

Reply:
Thank you for your comments.
Document updated with new requirements.